### ISSUE TITLE
ebpf cachestat creates empty charts when disabled

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#10786]
##### Bug report summary

ssia

ebpf cachestat is disabled by default

```cmd
[pc netdata]# grep "cachestat =" -B 1 ebpf.conf
[ebpf programs]
    cachestat = no
```

<img width="1275" alt="Screenshot 2021-03-17 at 13 34 54" src="https://user-images.githubusercontent.com/22274335/111454315-90ee2d00-8725-11eb-8bf7-0f6bde2721b4.png">

##### OS / Environment

```cmd
[pc netdata]# uname -a; grep -Hv "^#" /etc/*release
Linux pc 5.10.19-1-MANJARO #1 SMP PREEMPT Fri Feb 26 11:38:34 UTC 2021 x86_64 GNU/Linux
/etc/arch-release:Manjaro Linux
/etc/lsb-release:DISTRIB_ID=ManjaroLinux
/etc/lsb-release:DISTRIB_RELEASE=20.2.1
/etc/lsb-release:DISTRIB_CODENAME=Nibia
/etc/lsb-release:DISTRIB_DESCRIPTION="Manjaro Linux"
/etc/manjaro-release:Manjaro Linux
/etc/os-release:NAME="Manjaro Linux"
/etc/os-release:ID=manjaro
/etc/os-release:ID_LIKE=arch
/etc/os-release:BUILD_ID=rolling
/etc/os-release:PRETTY_NAME="Manjaro Linux"
/etc/os-release:ANSI_COLOR="32;1;24;144;200"
/etc/os-release:HOME_URL="https://manjaro.org/"
/etc/os-release:DOCUMENTATION_URL="https://wiki.manjaro.org/"
/etc/os-release:SUPPORT_URL="https://manjaro.org/"
/etc/os-release:BUG_REPORT_URL="https://bugs.manjaro.org/"
/etc/os-release:LOGO=manjarolinux
```

##### Netdata version

`netdata v1.29.3-155-g65f4f12e` 

##### Component Name

`collectors/ebpf`

##### Steps To Reproduce

1. install netdata
2. check `Applications->ebpf cachestat`

##### Expected behavior

ebpf cachestat doesn't create empty charts when disabled.

**Comment by ilyam8** [comment]
These charts

https://github.com/netdata/netdata/blob/65f4f12e8606a14d2c133569890e294275c2e056/collectors/ebpf.plugin/ebpf_cachestat.c#L266-L308

**Comment by thiagoftsm** [comment]
This will be fixed with the PR #10756
